gym_roboy/envs/ros_proxy.py

```diff
@@ -116,9 +116,9 @@ def _check_service_available_or_timeout(self, client) -> None:
             raise TimeoutError("ROS communication timed out")
 
     def read_state(self):
-        self._check_service_available_or_timeout(self.step_client)
+        self._check_service_available_or_timeout(self.observation_client)
         req = GymStep.Request()
```

Comment:
GymStep should contain now a mandatory "set_points", so we probably should use an empty or more simple Request message here. What do you think @BarisYazici.

Reply:
Yes. Let's make a new issue just for updating the message types